fuel-vm/src/interpreter/blockchain.rs

```diff
@@ -1211,12 +1100,9 @@ pub(crate) struct StateWriteWordCtx<'vm, S> {
     pub storage: &'vm mut S,
     pub memory: &'vm MemoryInstance,
     pub context: &'vm Context,
-    pub profiler: &'vm mut Profiler,
     pub new_storage_gas_per_byte: Word,
-    pub current_contract: Option<ContractId>,
     pub cgas: RegMut<'vm, CGAS>,
     pub ggas: RegMut<'vm, GGAS>,
-    pub is: Reg<'vm, IS>,
```

Comment:
Why are `current_contract` and `is` removed?

Reply:
It was used by profiler